service/integration/registered_resources_test.go

```diff
@@ -560,6 +565,39 @@ func (s *RegisteredResourcesSuite) Test_CreateRegisteredResourceValue_WithMetada
 	s.Require().Len(created.GetMetadata().GetLabels(), 2)
 }
 
+func (s *RegisteredResourcesSuite) Test_CreateRegisteredResourceValue_With_ActionAttributeValues_Succeeds() {
```

Comment:
Do you think it's worth it to add a multiples scenario or update this one to have two `ActionAttributeValue` combos?

Reply:
Good call, I think updating this test to have 2 is fine and further tests the `copyfrom` sqlc logic. Will add another.